### DISCUSSION TITLE
How to override query validation error message?

### DISCUSSION THREAD

**Discussion by polovyivan** [graphql-java/graphql-java#2866] [Q&A]
How do I override this type of validation error message? I want to translate it to another language depending on the user's location.

![image](https://user-images.githubusercontent.com/33532311/174498627-fec46621-630c-4b40-bf92-da4e941ef404.png)
![image](https://user-images.githubusercontent.com/33532311/174498645-ccfa48f5-eeda-47fe-8ed0-7188af3d9a56.png)

Below are logs from the console.

![image](https://user-images.githubusercontent.com/33532311/174498585-a7c49e8f-d8ea-4fe8-99cf-58282c8dd13c.png)

**Comment by bbakerman** [comment]
I am sorry to say that we have no mechanism to do this.  Ideally we should have a i18n mechanism built into graphql-java and we would construct messages using it.  However we dont have this. 

 I looked into a i18n mechanism a long time back - https://github.com/graphql-java/graphql-java/pull/1473

But the way the validation code "assembles" error messages is not amenable to i18n right now and hence that needs to be refactored a fair bit before we could do i18n properly

**Reply by dondonz** [reply]
Although we don't have i18n support yet, it is on the roadmap. We're actively working on it. Thanks for letting us know there is demand for the feature
